Contuity/Views/Jot/Write/WriteJotViewController.swift

```diff
@@ -20,10 +20,22 @@ class WriteJotViewController: UIViewController {
 
     @IBOutlet private (set) var textView: UITextView!
     @IBOutlet private (set) var saveButton: UIButton!
-
+    @IBOutlet private (set) var addAttributeButton: UIButton!
+    /// This varible keeps track of the FollowUpViewControllers created by this WriteJotViewController.
+    /// When a jot is written, the followups are also stored to the database.
+    private var followUpVCs: [FollowUpViewController] = []
```

Comment:
I tried to do that but, if I got a Date from the FollowUpVC's datepicker, it gives back the current date and time because the user has not selected the desired date yet. And I couldn't have a reference to a date that was being mutated by the datepicker, so I decided to hold all the followUpVCs which had datepickers and after the dates were finalized, I'd retrieve the dates. If you could figure out a way to do this, that'd be good, but this is the best I came up with.